index.html

```diff
@@ -0,0 +1,299 @@
+                        </ul>
+                        </div>
+        
+                    </div>
+                    <div class="project2">
+                        <div class="language-icon"><img src="./images/icon-skills (1).png" alt=""></div>
+                        <div class="lang-text">Skills</div>
+                        <ul class="lang-text-list">
+                            <li class="li-lang" >CodeKit</li>
+                            <li class="li-lang">Github</li>
+                            <li class="li-lang">Codepen</li>
+                            <li class="li-lang">Gitlab</li>
+                            <li class="li-lang">Terminal</li>
+                        </ul>
+                        </div>
+        
+                    </div>
+                    </div>
+                
+                </section>
+                </section>
+    
+                <section class="contact" id="contact"> 
+                
+                    <section class="form-section">
+                    
+                        <div class="form-description">
+                        <p class="touch">I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</p>
+                        </div>
+                    
+                        <div class="contact-form">
+                     <form action="https://formspree.io/f/mjvzwlqy" id="form" method="post">
+                       
+                        <ul>
+                            
+                            <li >
+                                <label for="name">
+                                    <input type="text" id="name" class="form-data"  name="user_name" placeholder="Full name" maxlength="30" required>
+                                </label>
+                            </li>
+                            <li>
+                                <label for="mail">
+                                    <input type="email" id="mail" class="form-data" name="user_email" placeholder="Email address" required>
+                                </label>
+        
+                            </li>
+                            <li>
+                                <label for="msg">
+                                    <textarea name="user_message" class="form-data" id="msg" cols="30" rows="5" maxlength="500" placeholder="Enter text here" required></textarea>
+                                </label>
+                            </li>
+                            <li><small class="error-message" id="errorMessage"></small></li>
+                            <li class="button back-image">
+                                <button type="submit" class="btn">Get in touch</button>
+                            </li>
+        
+                        </ul>
+        
+                    </form>
+        
+                        </div>
+                    </section>
+       
+                </section>
+
```

Comment:
- [ ] This section is duplicated, we have the same one online `242`, please remove one and keep the other.

![5](https://user-images.githubusercontent.com/77843050/213681857-fc71699e-6600-4476-aa73-b5cc827d292d.gif)